services/ui-src/src/components/export/FieldsSubsection.tsx

```diff
@@ -0,0 +1,108 @@
+import { useContext } from "react";
+import { Table, SpreadsheetWidget, ReportContext } from "components";
+import { Box, Heading } from "@chakra-ui/react";
+import {
+  pareseFieldData,
+  parseCustomHtml,
+  pdfPreviewTableNumberParse,
+} from "utils";
+import { sxDataTable } from "components/pages/Export/McparPdfExport";
```

Comment:
If this is coming from components let's put it in the component import, or else pass it in to the component directly rather than importing